common/i18n/resources/en.js

```diff
@@ -63,6 +63,7 @@ module.exports.translation = {
     primaryDesignatedManagementObjective: {
       title: 'Primary Designated Management Objective (1990 - 2020)',
       rowName: '',
+      conservation: 'Conservation',
```

Comment:
use designatedManagementObjective.biodiversityConservation and add it to all languages